plugins/system/core/classes/menu.php

```diff
@@ -254,9 +254,9 @@ private function dropdown($item) {
 		$this->menu .= '<div class="sp-dropdown-inner">';
 		$this->navigation($item, $firstitem, 0,  'sp-dropdown-items');
 
-		// only module exist
-		if (count($item->params->get('menulayout')) == 1) {
-			$mega_json = $item->params->get('menulayout');
+		$mega_json = $item->params->get('menulayout');
+		if ($mega_json)
```

Comment:
@nayemmajhar this value can be a non-array and still boolean equivalent to true, such as "{}”, “abc”, etc. This will cause issues again. Thats why I suggested adding the `is_array` check.